pombola/writeinpublic/views.py

```diff
@@ -185,8 +158,22 @@ def get_form_initial(self, step):
         else:
             return super(WriteInPublicNewMessage, self).get_form_initial(step=step)
 
+    def post(self, request, *args, **kwargs):
+        step = kwargs.get('step')
+        if step == 'preview':
```

Comment:
is this the right step to verify? Isn't there a submit step that would be more suitable?

Reply:
formtools calls `form.validate` from the [post](https://github.com/jazzband/django-formtools/blob/2.1/formtools/wizard/views.py#L268-L313) function and if it's valid, calls `render_next_step`. I could try and turn the captcha field into a custom form field. If we don't go that route, I think the recaptcha should be validated before the rest of the fields.

Reply:
Tried to create a form field for the Recaptcha in [this branch](https://github.com/OpenUpSA/pombola/tree/delena-write-to-mp-recaptcha-field), but formtools is caching the response or something.

Reply:
can I bypass the captcha by just doing a POST to the final step that sends? Or can I only send if i've gone through the preview step? What data is used by the final step? session data only, or user-provided input?

Put another way, if I modify the information in the preview, is the modified information sent?